Cargo.lock

```diff
@@ -10,15 +10,15 @@ dependencies = [
  "actix-rt",
  "actix_derive",
  "bitflags",
- "bytes",
+ "bytes 0.5.6",
```

Comment:
I suggest we don't go this long path (#3869) of trying to upgrade dependencies in this PR, so the quick fix here should be:

1. revert the Cargo.lock changes
2. run `cargo check`, so it only updates the relevant parts